spec/spec_helper.rb

```diff
@@ -1,37 +1,13 @@
-# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
-
-ENV['RAILS_ENV'] ||= 'test'
+if ENV['CI']
+  require 'simplecov'
+  SimpleCov.start
+end
 
-# Load the ManageIQ environment
-require File.expand_path("../manageiq/config/environment", __FILE__)
+require 'manageiq-graphql'
 
-require 'rspec/rails'
+Dir[ManageIQ::GraphQL::Engine.root.join("spec/support/**/*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
 
 RSpec.configure do |config|
-  config.expect_with :rspec do |expectations|
-    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
-  end
-
-  config.mock_with :rspec do |mocks|
-    mocks.verify_partial_doubles = true
-  end
-
-  config.shared_context_metadata_behavior = :apply_to_host_groups
-  config.filter_run_when_matching :focus
-  config.example_status_persistence_file_path = "spec/examples.txt"
-  config.disable_monkey_patching!
-  config.default_formatter = "doc" if config.files_to_run.one?
-  config.order = :random
-
-  config.infer_spec_type_from_file_location!
-  config.filter_rails_from_backtrace!
-  Kernel.srand config.seed
```

Comment:
This was the only thing I wasn't sure about.  I didn't know if you made these changes explicitly or they came from `rspec --init`.  I *think* most of this is already the default in rspec, so it's redundant and it isn't in any of the other repos including manageiq-api.

Reply:
Hmmm Yeah I think you're right here. In any case, we'll re-add what makes sense as we go.

(For example, I'd prefer we keep the `config.disable_monkey_patching!` above; that's not default in RSpec, but there's no reason we need to enable monkeypatching here).